sys/linux/dev_kvm.txt

```diff
@@ -232,12 +232,41 @@ kvm_text_x86_64 {
 	size	len[text, intptr]
 }
 
-kvm_text_arm64 {
+kvm_text_arm64 [
+	text_instr	kvm_text_arm64_instr
```

Comment:
This is true.
Perhaps we could at least save/restore registers around the ifuzz code.
But even running it as-is still looks better (from bug triggering capability) than running syzos and ifuzz strictly separably. If the generated syzos program will contain ifuzz chunk, then it's the same or better than running just ifuzz chunk.
Potentially we could make ifuzz always the last optional call. But I would start without that and just rely on coverage-guidance and good seeds.

Reply:
Ok, let's just start with what we have now (copy code to a separate page, call it and cross the fingers).
In the future we can probably analyze the control flow and e.g. prohibit backward branches to prevent infinite loops.

Reply:
Do we need to copy it? Can we execute it in-place?

Reply:
That's a cheap way to get it padded with RET instructions and surrounded by guard pages, nothing more.

Reply:
Fair enough, will do.